plugins/manual_classification/manual_classification_templates/manual_classification_template.html

```diff
@@ -3,64 +3,52 @@
 
 <head>
     <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='forms.css')}}" media="screen" />
-    <style type="text/css">
-        @import url("{{url_for('static', filename='forms.css')}}");
-
-        table {
-            font-family: arial, sans-serif;
-            border-collapse: collapse;
-            width: 100%;
-        }
-
-        td,
-        th {
-            border: 1px solid #dddddd;
-            text-align: left;
-            padding: 8px;
-        }
-
-        tr:nth-child(even) {
-            background-color: #dddddd;
-        }
-    </style><!-- TODO -->
 </head>
 
 <body>
     <div class="qhana-microfrontend">
         {% call forms.render_form() %}
         <!-- class name -->
         {{ forms.render_fields(schema, values=values, errors=errors) }}
-        <div class="qhana-form" style="flex-direction: row; flex-wrap: wrap;">
+        <div class="qhana-table" style="flex-direction: row; flex-wrap: wrap; 
+        height: 750px; overflow: auto;">
             <!-- select id's -->
             <table>
-                <tr>
+                <tr">
                     <th>Select</th>
                     {% for attr in attr_list%}
                     <th>{{attr}}</th>
                     {% endfor %}
-                </tr>
-                {% for entity in entity_list %}
-                <tr>
-                    <td class="qhana-form-field" style="width: 3em; padding: 1em;">
-                        <input type="checkbox" id={{entity["ID"]}} name={{entity["ID"]}}>
-                    </td>
-                    {% for attr in attr_list%}
-                    <td>{{entity[attr] if attr in entity.keys() else ""}}</td>
+                    </tr>
+                    {% for entity in entity_list %}
+                    <tr onclick="selectRow(this)">
+                        <td style="padding: 1em">
+                            <input class="qhana-form-input" style="height: 1.5em; width: 1.5em" type="checkbox"
+                                id={{entity["ID"]}} name={{entity["ID"]}}>
+                        </td>
+                        {% for attr in attr_list%}
+                        <td>{{entity[attr] if attr in entity.keys() else ""}}</td>
+                        {% endfor %}
+                    </tr>
                     {% endfor %}
-                </tr>
-                {% endfor %}
             </table>
-
         </div>
+        <br>
         <div class="qhana-form-buttons">
-            {{ forms.submit("validate")}}
+            {{ forms.submit("validate", target="microfrontend")}}
             {{ forms.submit("continue", action=process)}}
             {{ forms.submit("done", action=done)}}
         </div>
         {% endcall %}
     </div>
 
     <script src="{{url_for('static', filename='microfrontend.js')}}"></script>
+    <script type="text/javascript">
+        function selectRow(row) {
```

Comment:
Follow up from above: If a content security policy would be set by the server then the script tag would need to inlude a nonce (that should be generated new for each request) that is whitelisted in the CSP to allow the script to run. The selectRow function would have to be bound to the html elements using the `addEventListener` methods of the elements. (again, no changes needed her, just letting you know)